packages/app/src/components/uiBlocks/GroupDropDownItem.vue

```diff
@@ -1,7 +1,10 @@
 <template>
   <div
     class="group-item w-full flex flex-col"
-    :class="[isOpen && 'bg-gradient-to-b from-darkBlue to-darkViolet', !isOpen && 'bg-secondary']"
+    :class="[
+      isOpen && 'bg-gradient-to-b from-darkBlue to-darkViolet',
+      !isOpen && 'bg-secondary bg-opacity-50',
```

Comment:
I wasn't sure if you want to use here the orange gradient too

Reply:
no, that looks ugly